### ISSUE TITLE
Broken highlighting with ft=htmldjango

### ISSUE THREAD

**Issue by radimsuckr** [luochen1990/rainbow#105]
### **Describe the bug**
Rainbow parentheses breaks the Django tags highlighting.

### **To Reproduce**
Open any HTML file with Django template tags and make sure the filetype is set to `htmldjango`.

### **Expected behavior**
Disable rainbow parentheses for Django tags completely as nested parentheses don't occur in them.

### **Screenshots**
![image](https://user-images.githubusercontent.com/2447438/45649631-2fa16080-bacc-11e8-8f4b-6bfb5fa7bd69.png)

### **Additional context**
Syntax highlighting works as expected when I explicitly turn off rainbow parentheses for given buffer.

**Comment by luochen1990** [comment]
can you please provide a minimal complete example file which can reproduce your problem?

**Comment by radimsuckr** [comment]
The minimal example is as simple as `{% extends "base.html" %}`, which results into following screenshot.

![image](https://user-images.githubusercontent.com/2447438/45734134-a3825c80-bbe3-11e8-900c-a1ce7e54d460.png)

**Comment by luochen1990** [comment]
you can simply disable rainbow for this filetype.  Or if you know about RegExp in Vim, you can define your parentheses for htmldjango files yourself (maybe you will need zero-width matching).

**Comment by radimsuckr** [comment]
Yeah, disabling it is an option, but I was hoping that maybe it'd be possible to have HTML rainbow parentheses combined with Django tags.
